listenbrainz_spark/stats/user/entity.py

```diff
@@ -106,10 +106,19 @@ def create_messages(data, entity: str, stats_range: str, from_date: datetime, to
         to_date: The end time of the stats
         message_type: used to decide which handler on LB webserver side should
             handle this message. can be "user_entity" or "year_in_music_top_stats"
+        database: the name of the database in which the webserver should store the data
 
     Returns:
         messages: A list of messages to be sent via RabbitMQ
     """
+    if database is None:
+        database = f"{entity}_{stats_range}"
+
+    yield {
```

Comment:
If the couch DB creation fails, then we'll have a ton of failed insertion messages and presumably loads of sentry messages. I think we need to be more careful about the possible failures to create DB.

Reply:
Agree that we should be more careful for failures to create db. Regarding number of sentry messages, see other comment.